app/Auth0/Auth0ClusterSDK.php

```diff
@@ -32,4 +32,11 @@ public function createClientsForIntegration(Integration $integration): array
             )
         );
     }
+
+    public function createClientsForIntegrationOnAuth0Tenant(
+        Integration $integration,
+        Auth0Tenant $auth0Tenant
+    ): Auth0Client {
+        return $this->auth0TenantSDKs[$auth0Tenant->value]->createClientForIntegration($integration);
```

Comment:
Note that in theory it's possible that there is no SDK set for a given tenant, for example when the config is left blank for that tenant. Is it okay for the script to just crash on that, under the assumption that you should only run it with non-empty config values for Auth0 since you explicitly do it to create clients? And/or, is it okay for this more generic method to ignore this potential error and leave it to the caller to deal with that?

Reply:
I think it's best to throw an error then, something like `Auth0TenantNotConfigured`?